src/compiler/checker.ts

```diff
@@ -20803,6 +20802,21 @@ namespace ts {
             }
         }
 
+        function errorSubsequentVariableDeclarationMustHaveSameType(firstDeclaration: Declaration, firstType: Type, nextDeclaration: Declaration, subsequentType: Type): void {

```

Comment:
I'd use 'Next' consistently for the function name and for -Type and -Declaration. It's easier to type than subsequent.